app/src/main/java/at/roteskreuz/stopcorona/model/repositories/DashboardRepository.kt

```diff
@@ -2,48 +2,26 @@ package at.roteskreuz.stopcorona.model.repositories
 
 import android.content.SharedPreferences
 import at.roteskreuz.stopcorona.constants.Constants
-import at.roteskreuz.stopcorona.model.db.dao.NearbyRecordDao
 import at.roteskreuz.stopcorona.skeleton.core.utils.booleanSharedPreferencesProperty
-import at.roteskreuz.stopcorona.utils.asDbObservable
-import io.reactivex.Observable
 
 /**
  * Repository for managing dashboard content.
  */
 interface DashboardRepository {
 
-    val showMicrophoneExplanationDialog: Boolean
-
-    /**
-     * Observes the number of met people.
-     */
-    fun observeSavedEncountersNumber(): Observable<Int>
-
     /**
-     * Do not show explanation dialog again.
+     * Information if the automatic handshake was enabled automatically on the first start
```

Comment:
I don´t get this, we don´t have a service anymore...

Reply:
Renamed in new commit